mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -15,7 +15,10 @@
 #include "mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h"
 #include "mlir/Dialect/Rock/Tuning/RockTuning.h"
 #include "mlir/Dialect/Rock/utility/AmdArchDb.h"
+#include "mlir/Dialect/Rock/utility/math.h"
 #include "llvm/ADT/SmallString.h"
+#include <algorithm>
```

Comment:
Do we need this? Can you migrate to one of the `llvm::foo`s that takes a `Range` instead?

Reply:
Yes, I used `llvm::for_each` which uses ranges